packages/design-system-react/package.json

```diff
@@ -48,14 +48,24 @@
   },
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
+    "@storybook/react": "^8.3.5",
+    "@storybook/test": "^8.3.5",
     "@types/jest": "^27.4.1",
+    "@types/react": "^18.2.0",
+    "@types/react-dom": "^18.2.0",
     "deepmerge": "^4.2.2",
     "jest": "^27.5.1",
     "ts-jest": "^27.1.4",
     "typedoc": "^0.24.8",
     "typedoc-plugin-missing-exports": "^2.0.0",
     "typescript": "~5.2.2"
   },
+  "peerDependencies": {
+    "@metamask/design-system-tailwind-preset": "^0.0.0",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
```

Comment:
Aligns with minimum react version that we want to support
- extension ["react": "^16.12.0"](https://github.com/MetaMask/metamask-extension/blob/develop/package.json#L406)
- portfolio ["react": "^18.2.0"](https://github.com/consensys-vertical-apps/metamask-portfolio/blob/main/package.json#L65)